### PR TITLE
fix - Cr 155 Development build and verify "CV Creole" FTM

### PR DESCRIPTION
**Pull request by Rajesh1041**
# Changes
- caboverdecreole language folder(build and lang folder)

# How to test
- Select the caboverdecreole from drop down to check content

Ref: [CR-155](https://curiouslearning.atlassian.net/jira/software/projects/CR/boards/7?selectedIssue=CR-155)

[CR-155]: https://curiouslearning.atlassian.net/browse/CR-155?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# FeedTheMonsterJS
+# FeedTheMonsterJS backup dev
```
Comment (ashwinnair-chimple) [inline README.md:1]: please remove backup dev

Comment (Rajesh1041) [reply README.md]: Removed!

**Review by ashwinnair-chimple — APPROVED** [review]
Approved

File: build/feedTheMonster.js
Comment (ashwinnair-chimple) [inline build/feedTheMonster.js:1]: @Rajesh1041 feedthemonster.js Build file is empty please check this

Comment (Rajesh1041) [reply build/feedTheMonster.js:1]: I think it should be fixed now,since you have updated this file.

**Pull request merged**